src/rpc/rpc_message.js

```diff
@@ -21,7 +21,7 @@ const RPC_VERSION_NUMBER = Buffer.from([
 // possible in order to ensure complaince on both sides of the RPC call.
 function clone_customizer(value) {
     // Mongo object ids are always serialized into strings and never deserialized back.
-    if (mongo_utils.is_object_id(value)) {
+    if (db_client.instance().is_object_id(value)) {
```

Comment:
Moved back to mongo_utils because of FE builds failing on the pg-native package (since this code requires Postgres).
Did not want to add the package to the FE because we will move from ObjectId to string and this doesn't make sense to include packages to FE (which currently supports MongoDB).
It will be removed on the move to string anyway.